src/riak_ql_lexer.xrl

```diff
@@ -46,6 +46,7 @@ REGEX = (/[^/][a-zA-Z0-9\*\.]+/i?)
 
 IDENTIFIER = ([a-zA-Z][a-zA-Z0-9_\-]*)
 QUOTED_IDENTIFIER = \"(\"\"|[^\"\n])*\"
+COMMENT_MULTILINE = (/\*([^*]|[\r\n]|(\*+([^*/]|[\r\n])))*\*+/)|(--.*)
```

Comment:
I could have it capture the comments with a slightly simplified `(/\*([^*]|[\r\n]|(\*([^/]|[\r\n])))*\*/)|(--.*)`.